.github/workflows/build.yml

```diff
@@ -157,6 +157,15 @@ jobs:
             gusto-repo/integration-tests/transport/test_embedded_dg_advection.py
         timeout-minutes: 10
 
+      - name: Run spyro smoke tests
+        if: (success() || steps.install.conclusion == 'success') && matrix.arch == 'default'
+        run: |
+          . venv/bin/activate
+          git clone --depth 1 https://github.com/NDF-Poli-USP/spyro.git spyro-repo
+          pip install --verbose ./spyro-repo
+          mpiexec -n 6 python -m pytest spyro-repo/test_integration/
```

Comment:
Are all of your tests marked as `pytest.mark.parallel(6)`? I would expect this to look like
```
mpiexec -n 6 python -m pytest spyro-repo/test_integration/ -m parallel[6]
```

Reply:
They aren't market. I can do this and merge into spyro main as soon as our runner is back in operation

Reply:
Changed it. Now I am just waiting for the spyro runner to finish the tests before merging there